html/apps/forgotpassword/js/passwordrecovery.js

```diff
@@ -2,50 +2,50 @@ $(document).ready(function()
 {
     var emailerr = false, session = true;
 
-    $('#submit').click(function(){
+    function resetFields(){
+        $("#status-box").text("");
+        $("#status-box").removeClass();
+        $("#email").val("");
+        $("#email").removeClass();
+        $("#emailfield").text("");
+        $("#emailfield").removeClass();
+    }
+
+    function checkEmpty(){
+        var emailval = $("#email").val();
+        if(!emailval && !($("#email").is(":focus"))){
+            $("#emailfield").text("Required field!");
+            $("#email").removeClass();
+            $("#emailfield").removeClass();
+            $("#email").addClass("error");
+            $("#emailfield").addClass("error");
+            emailerr = true;
+        } else {
+            emailerr = false;
+            $("#email").removeClass();
+            $("#emailfield").removeClass();
+            $("#emailfield").text("");
+        }
+    }
+
+    $("#submit").click(function(){
         checkEmpty();
         if(emailerr){
-            $('#status-box').text("Please fill email");
+            $("#status-box").text("Please fill email");
         } else{
```

Comment:
Follow the codacy matrix standard and fix the indentation.
instead of " } else{ " correct it as " } else { "